luxonis_ml/data/datasets/luxonis_dataset.py

```diff
@@ -225,6 +225,8 @@ def _load_df_offline(self, sync_mode: bool = False) -> Optional[pd.DataFrame]:
             annotations_path = self.annotations_path
         else:
             annotations_path = osp.join(self.tmp_dir, "annotations")
+        if not osp.exists(annotations_path):
+            return None
```

Comment:
Should we `return 0` here? Or if not change the return type to `Optional[int]`

Reply:
Not sure what you mean. The function is annotated as `Optional[pd.DataFrame]`.

Reply:
My mistake - nevermind